### ISSUE TITLE
Cannot open xopp files

### ISSUE THREAD

**Issue by stefan31415** [xournalpp/xournalpp#6157]
### Operating System

Linux

### (Linux only) Distribution

XUbuntu 24.10"

### (Linux only) Desktop Environment

xfce

### (Linux Only) Display Server

None

### Installation Method

apt

### Xournal++ Version

1.2.3

### libgtk Version

 3.24.43

### Bug Description

Do not open the file .xopp, display  something like " sorry, cannot open remote file, please copy the file localy..."

### Expected Behaviour

Open the file ...

### Steps to Reproduce

open any *.xopp file or pdf having .xopp

**Comment by rolandlo** [comment]
From what folder are you trying to open a .xopp file?

**Comment by stefan31415** [comment]
Hi Roland,

Thank you for your attention.
The original folder was located on a mounted disk: /media/spi/Data1.4T1/Lexar/REPARATIONS/FLORENNES/ROSEE/Rosée SPRL/Conso elect 2021-2023/.

I tested it on my main disk: /home/spi/Bureau/POUB/.

the tested file (without the pdf having the similar name) : [schéma Houet - annoté par SPI.xopp.zip](https://github.com/user-attachments/files/18129553/schema.Houet.-.annote.par.SPI.xopp.zip)
Regards,

Stefan

**Comment by rolandlo** [comment]
The .xopp file you uploaded still refers to the PDF background on the mounted disk: `/media/spi/Lexar/REPARATIONS/FLORENNES/ROSEE/Rosée SPRL/RGIE/2021 photovoltaique/schéma Houet.pdf`

That appears to be the issue.

**Comment by stefan31415** [comment]
You're right, I moved the files. "/media/spi/Lexar" was a USB key, and I copied the contents to a hard drive: /media/spi/Data1.4T1/Lexar/.

However, this behavior isn't typical; when moving files, it usually prompts you to select a different source upon opening.
![capture1](https://github.com/user-attachments/assets/adc47f48-8492-47ff-829d-e724a8b4d927)

In this case, it only displays a message, and there are no actions available.
![capture2](https://github.com/user-attachments/assets/a9d3c0d7-4a4d-4461-801c-146488a37d1b)

P.S.: I always place the xopp file with the background file in the same folder.

**Comment by stefan31415** [comment]
I installed the version "xournalpp-1.2.5-Ubuntu-noble-x86_64.deb". The message has changed (now with more details), but there is still no action available.
![capture3](https://github.com/user-attachments/assets/70004c72-7371-4a20-89a5-f45148277161)

I conducted a test (test1.xopp) with a new file placed in the same directory, then moved it to a subdirectory: everything works, and it asks where the source is located.

So the path does not seem to be the issue.

**Comment by stefan31415** [comment]
The issue has been resolved.
There was a rogue directory "/media/spi/Lexar" owned by root. 
Running "sudo rm -d /media/spi/Lexar" fixed the problem. 

Thank you for your assistance.

**Comment by eldipa** [comment]
@rolandlo when you have time, please close the issue. It seems that it was resolved.

1 issue done, 1.1k left  :D
